### ISSUE TITLE
JPM isn't creating update.rdf

### ISSUE THREAD

**Issue by pdehaan** [meandavejustice/min-vid#23]
Not sure why, but when I run `npm run package` locally, it generates the XPI, but I don't see an update.rdf (which we'll need for auto updates via Test Pilot, or whatever).

**Comment by meandavejustice** [comment]
we need to provide this [key](https://github.com/mozilla/testpilot/blob/master/addon/package.json#L19) in order for jpm to generate an `update.rdf`

**Comment by wresuolc** [comment]
Blocked on #13

**Comment by meandavejustice** [comment]
This url will more than likely look like: `https://testpilot.cdn.mozilla.net/minvid/min_vid-1.0.0-an+fx.xpi``

**Comment by pdehaan** [comment]
@wresuolc When you copy these files over to the testpilot CDN, do you rename the files or leave them with the version number?

Here's our current config for Activity Stream (which uses static filenames for the latest XPI and RDF):

```js
"updateLink": "https://moz-activity-streams.s3.amazonaws.com/dist/activity-streams-latest.xpi",
"updateURL": "https://moz-activity-streams.s3.amazonaws.com/dist/update.rdf",
```

**Comment by wresuolc** [comment]
I've been leaving them as-is

**Comment by meandavejustice** [comment]
fixed in #171
